lib/teleterm/daemon/config.go

```diff
@@ -37,9 +37,12 @@ type Config struct {
 	// Electron app. This is to ensure that the server public key is written to the disk under the
 	// expected location by the time we get around to creating the client.
 	CreateTshdEventsClientCredsFunc CreateTshdEventsClientCredsFunc
-	GatewayCertReissuer             *GatewayCertReissuer
 	// PrehogAddr is the URL where prehog events should be submitted.
 	PrehogAddr string
+	// TSHDEventsClient is a client to send events to the Electron App. This client is usually
+	// formed during gateway creation, but can also be supplied directly during service
+	// initialization here.
```

Comment:
Same as above, I don't think the part about the client being initialized during gateway creation is correct.